tools/build_variables.bzl

```
@@ -314,7 +314,26 @@ core_sources_full = core_sources_full_mobile + [
 
 libtorch_core_sources = sorted(core_sources_common + core_sources_full + core_trainer_sources)
 
-libtorch_distributed_sources = [
+libtorch_distributed_windows_sources = [
```

Nit: Please consider renaming `libtorch_distributed_windows_sources` to `libtorch_distribtued_base_sources`, as there are nothing windows specific about them
```suggestion
libtorch_distributed_base_sources = [
```

---

I called them `windows` because they're the only sources that are safe to build under Windows. I'll rename them to `base` as suggested, and add a comment to clarify why we need to separate them.